tools/scripts-config-eslint/package.json

```diff
@@ -19,8 +19,8 @@
   "dependencies": {
     "@babel/eslint-parser": "^7.18.9",
     "@talend/eslint-plugin": "^1.0.0",
-    "@typescript-eslint/parser": "^5.30.7",
-    "@typescript-eslint/eslint-plugin": "^5.30.7",
+    "@typescript-eslint/parser": "^5.48.0",
+    "@typescript-eslint/eslint-plugin": "^5.48.0",
```

Comment:
upgrade this to avoid the error message saying we are using a newer typescript version which is not supported by  the parser